src/Website/Services/SiteNews/InMemorySiteNewsProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <copyright file="InMemorySiteNewsProvider.cs" company="Clicker Heroes Tracker">
+// Copyright (c) Clicker Heroes Tracker. All rights reserved.
+// </copyright>
+
+namespace Website.Services.SiteNews
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// In memory provider for site news.
+    /// This is used when there is no azure table storage setup.
+    /// </summary>
+    public class InMemorySiteNewsProvider : ISiteNewsProvider
+    {
+        private readonly IDictionary<DateTime, IList<string>> siteNewsDictionary = new Dictionary<DateTime, IList<string>>();
```

Comment:
yeah true, i'll lock for correctness.  Of course this will not work in a multi machine environment ;)